integration/client/image_test.go

```diff
@@ -19,6 +19,7 @@ package client
 import (
 	"context"
 	"fmt"
+	. "github.com/containerd/containerd/defaults"
```

Comment:
`github.com/containerd/containerd/defaults` should not be placed in the standard library package, in addition, there is no need to use **.** to import the defaults package